Traceback (most recent call last):
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus\browser_decorator.py", line 201, in run_task
    result = func(driver, data)
  File "c:\Users\Admin\Downloads\Code\OOPFinalProject\Project\MyTest\try_surfing.py", line 22, in download_html_pages
    driver.google_get(link, bypass_cloudflare=True)
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\driver.py", line 2230, in google_get
    self.get_via(
    ~~~~~~~~~~~~^
        link,
        ^^^^^
    ...<4 lines>...
        timeout=timeout,
        ^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\driver.py", line 2199, in get_via
    self._tab = self._browser.get(link, referrer=referer, )
                ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\core\browser.py", line 310, in get
    frame_id, _, *_ = connection.send(cdp.page.navigate(url, referrer=referrer))
                      ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\core\connection.py", line 290, in send
    return self.perform_send(cdp_obj, _is_update, wait_for_response)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\core\connection.py", line 311, in perform_send
    result = parse_response(result, cdp_obj)
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\core\connection.py", line 50, in parse_response
    raise ChromeException(response["error"])
botasaurus_driver.exceptions.ChromeException: Invalid parameters [code: -32602]
